@kiva/kv-components/vue/KvChip.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+	<div class="tw-inline-flex">
```

Comment:
To fix the alignment of the X and cleanup your dom I would remove your extraneous div's. I think the layout should be: 
```
<button>
<div></div>
<kv-material-icon/>
</button>
```